CMakeLists.txt

```diff
@@ -12,8 +12,8 @@ project(mim_control)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 
-# Some additionnal optimization for the code.
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -funroll-loops -fPIC")
+# Some additionnal optimization for the code. set(CMAKE_CXX_FLAGS_RELEASE
+# "${CMAKE_CXX_FLAGS_RELEASE} -funroll-loops -fPIC")
```

Comment:
If you think this should be removed, please remove the comments instead of commenting out the lines.

Reply:
I updated the description, I hope it helps.

I commented it because I had a problem with release_mode but commenting this line didn't help. @MaximilienNaveau do you think it should be removed? I can keep it if no one has a problem with the release_mode.